build/scripts/build.ps1

```diff
@@ -275,24 +276,16 @@ function Deploy-VsixViaTool() {
 
     Write-Host "Uninstalling old Roslyn VSIX"
 
-    # Reverse the extension list so we uninstall in the proper order so that dependencies line up
```

Comment:
Why does it matter if there's a MEF cache rebuild in the middle? Nothing is loading it, and the cache will get blown the next time an install happens again.